staging/src/k8s.io/apiserver/pkg/server/options/server_run_options.go

```diff
@@ -143,9 +146,30 @@ func (s *ServerRunOptions) Validate() []error {
 		errors = append(errors, fmt.Errorf("--max-resource-write-bytes can not be negative value"))
 	}
 
+	if err := validateHSTSHeader(s.HSTSHeaders); err != nil {
+		errors = append(errors, err)
+	}
 	return errors
 }
 
+func validateHSTSHeader(hstsHeaders []string) error {
+	// HSTS Headers format: Strict-Transport-Security:max-age=expireTime [;includeSubDomains] [;preload]
```

Comment:
link to spec as well please. this one, right? https://tools.ietf.org/html/rfc6797#section-6.1